public/javascript/charCardContrsuctor.js

```diff
@@ -0,0 +1,11 @@
+const constrcutCharCard = character => {
+    let isCartoon = ''
+    character.cartoon ? isCartoon = 'Cartoon' : isCartoon = 'Non-Cartoon'
```

Comment:
Una cosa que no os hemos contado de los ternarios es que "retornan"...

```suggestion
    isCartoon = character.cartoon ? 'Cartoon' : 'Non-Cartoon'
```

Eso funciona igual ;)